.github/workflows/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+    schedule:
+      # Run everyday at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
+      - cron: "0 9 * * *"
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
and `develop`.

Reply:
i don't know what you mean.

Reply:
on:
  pull_request:
    push:
      branches:
        - 'release-*'
        - 'develop'
        - 'staging'
        - 'main'